hepcrawl/spiders/desy_spider.py

```diff
@@ -192,82 +168,66 @@ def crawl_s3_bucket(self):
                     )
 
     def start_requests(self):
-        """List selected folder locally or bucket on s3 and yield files."""
-
-        if self.s3_enabled:
-            requests = self.crawl_s3_bucket()
-        else:
-            requests = self.crawl_local_directory()
+        """List selected bucket on s3 and yield files."""
+        requests = self.crawl_s3_bucket()
 
         for request in requests:
             yield request
 
-    @staticmethod
-    def _has_to_be_downloaded(current_url):
-        def _is_local_path(url):
-            parsed_url = urllib.parse.urlparse(url)
-            return not parsed_url.scheme
+    @classmethod
+    def _is_local_path(cls, url):
+        parsed_url = urllib.parse.urlparse(url)
+        return not parsed_url.scheme
 
-        return _is_local_path(current_url)
+    @classmethod
+    def _has_to_be_downloaded(cls, current_url):
```

Comment:
what's the point of this? `_has_to_be_downloaded` is the same function as `_is_local_path`. Do we really need both?

Reply:
Actually I forgot to remove it, it's not used anumore. Thanks :+1: